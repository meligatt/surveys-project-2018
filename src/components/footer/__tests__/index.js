import React from 'react';
import Footer from '../index';

describe('Footer', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <Footer />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
