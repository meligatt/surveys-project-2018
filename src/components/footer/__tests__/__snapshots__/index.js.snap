// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="footer__container"
>
        <div
                className="footer__author"
        >
                <span>
                        Code challenge for Culture Amp - 2018 by Melissa Gattoni
                </span>
                <img
                        alt="logo"
                        height="40"
                        src="ahsoka.svg"
                        width="auto"
                />
        </div>
        <ul
                className="footer__link-list"
        >
                <li
                        className="footer__list-item"
                >
                        <a
                                href="https://www.linkedin.com/in/melissa-gattoni-54975b49/"
                                rel="noopener noreferrer"
                                target="_blank"
                        >
                                LinkedIn
                        </a>
                </li>
                <li
                        className="footer__list-item"
                        rel="noopener noreferrer"
                        target="_blank"
                >
                        <a
                                href="https://github.com/meligatt"
                        >
                                Github
                        </a>
                </li>
                <li
                        className="footer__list-item"
                        rel="noopener noreferrer"
                        target="_blank"
                >
                        <a
                                href="https://twitter.com/meligatt"
                        >
                                Twitter
                        </a>
                </li>
        </ul>
</div>,
      "className": "footer",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="footer__author"
>
            <span>
                        Code challenge for Culture Amp - 2018 by Melissa Gattoni
            </span>
            <img
                        alt="logo"
                        height="40"
                        src="ahsoka.svg"
                        width="auto"
            />
</div>,
          <ul
            className="footer__link-list"
>
            <li
                        className="footer__list-item"
            >
                        <a
                                    href="https://www.linkedin.com/in/melissa-gattoni-54975b49/"
                                    rel="noopener noreferrer"
                                    target="_blank"
                        >
                                    LinkedIn
                        </a>
            </li>
            <li
                        className="footer__list-item"
                        rel="noopener noreferrer"
                        target="_blank"
            >
                        <a
                                    href="https://github.com/meligatt"
                        >
                                    Github
                        </a>
            </li>
            <li
                        className="footer__list-item"
                        rel="noopener noreferrer"
                        target="_blank"
            >
                        <a
                                    href="https://twitter.com/meligatt"
                        >
                                    Twitter
                        </a>
            </li>
</ul>,
        ],
        "className": "footer__container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <span>
                Code challenge for Culture Amp - 2018 by Melissa Gattoni
</span>,
              <img
                alt="logo"
                height="40"
                src="ahsoka.svg"
                width="auto"
/>,
            ],
            "className": "footer__author",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Code challenge for Culture Amp - 2018 by Melissa Gattoni",
              },
              "ref": null,
              "rendered": "Code challenge for Culture Amp - 2018 by Melissa Gattoni",
              "type": "span",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "logo",
                "height": "40",
                "src": "ahsoka.svg",
                "width": "auto",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <li
                className="footer__list-item"
>
                <a
                                href="https://www.linkedin.com/in/melissa-gattoni-54975b49/"
                                rel="noopener noreferrer"
                                target="_blank"
                >
                                LinkedIn
                </a>
</li>,
              <li
                className="footer__list-item"
                rel="noopener noreferrer"
                target="_blank"
>
                <a
                                href="https://github.com/meligatt"
                >
                                Github
                </a>
</li>,
              <li
                className="footer__list-item"
                rel="noopener noreferrer"
                target="_blank"
>
                <a
                                href="https://twitter.com/meligatt"
                >
                                Twitter
                </a>
</li>,
            ],
            "className": "footer__link-list",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  href="https://www.linkedin.com/in/melissa-gattoni-54975b49/"
                  rel="noopener noreferrer"
                  target="_blank"
>
                  LinkedIn
</a>,
                "className": "footer__list-item",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "LinkedIn",
                  "href": "https://www.linkedin.com/in/melissa-gattoni-54975b49/",
                  "rel": "noopener noreferrer",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": "LinkedIn",
                "type": "a",
              },
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  href="https://github.com/meligatt"
>
                  Github
</a>,
                "className": "footer__list-item",
                "rel": "noopener noreferrer",
                "target": "_blank",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Github",
                  "href": "https://github.com/meligatt",
                },
                "ref": null,
                "rendered": "Github",
                "type": "a",
              },
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  href="https://twitter.com/meligatt"
>
                  Twitter
</a>,
                "className": "footer__list-item",
                "rel": "noopener noreferrer",
                "target": "_blank",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Twitter",
                  "href": "https://twitter.com/meligatt",
                },
                "ref": null,
                "rendered": "Twitter",
                "type": "a",
              },
              "type": "li",
            },
          ],
          "type": "ul",
        },
      ],
      "type": "div",
    },
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="footer__container"
>
          <div
                    className="footer__author"
          >
                    <span>
                              Code challenge for Culture Amp - 2018 by Melissa Gattoni
                    </span>
                    <img
                              alt="logo"
                              height="40"
                              src="ahsoka.svg"
                              width="auto"
                    />
          </div>
          <ul
                    className="footer__link-list"
          >
                    <li
                              className="footer__list-item"
                    >
                              <a
                                        href="https://www.linkedin.com/in/melissa-gattoni-54975b49/"
                                        rel="noopener noreferrer"
                                        target="_blank"
                              >
                                        LinkedIn
                              </a>
                    </li>
                    <li
                              className="footer__list-item"
                              rel="noopener noreferrer"
                              target="_blank"
                    >
                              <a
                                        href="https://github.com/meligatt"
                              >
                                        Github
                              </a>
                    </li>
                    <li
                              className="footer__list-item"
                              rel="noopener noreferrer"
                              target="_blank"
                    >
                              <a
                                        href="https://twitter.com/meligatt"
                              >
                                        Twitter
                              </a>
                    </li>
          </ul>
</div>,
        "className": "footer",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="footer__author"
>
              <span>
                            Code challenge for Culture Amp - 2018 by Melissa Gattoni
              </span>
              <img
                            alt="logo"
                            height="40"
                            src="ahsoka.svg"
                            width="auto"
              />
</div>,
            <ul
              className="footer__link-list"
>
              <li
                            className="footer__list-item"
              >
                            <a
                                          href="https://www.linkedin.com/in/melissa-gattoni-54975b49/"
                                          rel="noopener noreferrer"
                                          target="_blank"
                            >
                                          LinkedIn
                            </a>
              </li>
              <li
                            className="footer__list-item"
                            rel="noopener noreferrer"
                            target="_blank"
              >
                            <a
                                          href="https://github.com/meligatt"
                            >
                                          Github
                            </a>
              </li>
              <li
                            className="footer__list-item"
                            rel="noopener noreferrer"
                            target="_blank"
              >
                            <a
                                          href="https://twitter.com/meligatt"
                            >
                                          Twitter
                            </a>
              </li>
</ul>,
          ],
          "className": "footer__container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <span>
                  Code challenge for Culture Amp - 2018 by Melissa Gattoni
</span>,
                <img
                  alt="logo"
                  height="40"
                  src="ahsoka.svg"
                  width="auto"
/>,
              ],
              "className": "footer__author",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Code challenge for Culture Amp - 2018 by Melissa Gattoni",
                },
                "ref": null,
                "rendered": "Code challenge for Culture Amp - 2018 by Melissa Gattoni",
                "type": "span",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "logo",
                  "height": "40",
                  "src": "ahsoka.svg",
                  "width": "auto",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <li
                  className="footer__list-item"
>
                  <a
                                    href="https://www.linkedin.com/in/melissa-gattoni-54975b49/"
                                    rel="noopener noreferrer"
                                    target="_blank"
                  >
                                    LinkedIn
                  </a>
</li>,
                <li
                  className="footer__list-item"
                  rel="noopener noreferrer"
                  target="_blank"
>
                  <a
                                    href="https://github.com/meligatt"
                  >
                                    Github
                  </a>
</li>,
                <li
                  className="footer__list-item"
                  rel="noopener noreferrer"
                  target="_blank"
>
                  <a
                                    href="https://twitter.com/meligatt"
                  >
                                    Twitter
                  </a>
</li>,
              ],
              "className": "footer__link-list",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <a
                    href="https://www.linkedin.com/in/melissa-gattoni-54975b49/"
                    rel="noopener noreferrer"
                    target="_blank"
>
                    LinkedIn
</a>,
                  "className": "footer__list-item",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "LinkedIn",
                    "href": "https://www.linkedin.com/in/melissa-gattoni-54975b49/",
                    "rel": "noopener noreferrer",
                    "target": "_blank",
                  },
                  "ref": null,
                  "rendered": "LinkedIn",
                  "type": "a",
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <a
                    href="https://github.com/meligatt"
>
                    Github
</a>,
                  "className": "footer__list-item",
                  "rel": "noopener noreferrer",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Github",
                    "href": "https://github.com/meligatt",
                  },
                  "ref": null,
                  "rendered": "Github",
                  "type": "a",
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <a
                    href="https://twitter.com/meligatt"
>
                    Twitter
</a>,
                  "className": "footer__list-item",
                  "rel": "noopener noreferrer",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Twitter",
                    "href": "https://twitter.com/meligatt",
                  },
                  "ref": null,
                  "rendered": "Twitter",
                  "type": "a",
                },
                "type": "li",
              },
            ],
            "type": "ul",
          },
        ],
        "type": "div",
      },
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
